graderx/graders/manager.py

```diff
@@ -131,80 +156,97 @@ def compressed_results(course_name, lab):
     with open(file_path) as f:
-     data = json.load(f)
-     
+        data = json.load(f)
+
     return data
 
-def get_all_courses_data(only_stdout = False):
+
+def get_all_courses_data(only_stdout=False):
     all_courses = get_courses_config()
     if only_stdout:
-        all_courses = {course_name:course_data for course_name,course_data in all_courses.items() if course_data['type'] == 'stdout'}
+        all_courses = {course_name: course_data for course_name,
+                       course_data in all_courses.items() if course_data[COURSE_TYPE] == 'stdout'}
     for course_id, course_data in all_courses.items():
-        course_data['name'] = course_id
+        course_data[COURSE_NAME] = course_id
     return list(all_courses.values())
 
+
 def get_all_labs_data(course_id):
-    course_labs = get_course_data(course_id)['labs']
+    course_labs = get_course_data(course_id)[COURSE_LABS]
     for index, lab in enumerate(course_labs):
-        course_labs[index]["test_cases"] = stdout_common.get_test_cases(course_id, lab["name"])
+        course_labs[index][LAB_TEST_CASES] = stdout_common.get_test_cases(
+            course_id, lab[LAB_NAME])
     return course_labs
 
+
 def get_submission_files(course, lab, submission_id):
     course_grader = select_course_grader(course)
-    submission_files_paths = course_grader.get_submission_files(course, lab, submission_id)
+    submission_files_paths = course_grader.get_submission_files(
+        course, lab, submission_id)
     if submission_files_paths:
         return list(map(lambda path: path.name, submission_files_paths))
     else:
         raise SubmissionNotFoundError
 
+
 def update_submission_files(course, lab, submission_id, submission_files):
     course_grader = select_course_grader(course)
     try:
-        course_grader.update_submission_files(course, lab, submission_id, submission_files)
+        course_grader.update_submission_files(
+            course, lab, submission_id, submission_files)
     except FileNotFoundError:
         raise SubmissionNotFoundError
 
+
 def get_submission_file_content(course, lab, submission_id, file_name):
     course_grader = select_course_grader(course)
     try:
-        submission_file_content = course_grader.get_submission_file_content(course, lab, submission_id, file_name)
+        submission_file_content = course_grader.get_submission_file_content(
+            course, lab, submission_id, file_name)
         return submission_file_content
     except FileNotFoundError:
         raise SubmissionFileNotFoundError
 
+
 def get_submissions_list(course, lab):
     course_grader = select_course_grader(course)
     submissions_list = course_grader.get_not_fullmark_submissions(course, lab)
     return submissions_list
 
+
 def get_course_data(course_id):
     try:
         course_data = get_courses_config()[course_id]
     except KeyError:
         raise CourseNotFoundError()
-    course_data['name'] = course_id
+    course_data[COURSE_NAME] = course_id
```

Comment:
I think this should be something like `NAME_KEY`, since `COURSE_NAME` conveys the meaning that it's an actual course name.

Making a `CourseData` class that is constructed with `course_data` map and constructs an object out of it is the best solution though. But that can be done later